definitions/npm/redux-reactors_v1.x.x/test_redux-reactors_v1.x.x.js

```diff
@@ -0,0 +1,117 @@
+// @flow
```

Comment:
Move the test file under the flow-specific version.

Reply:
Done.